validators/terravaloper150w5e8tjk20nfafjzfhyht0aes8a5qr7meh8lg/profile.json

```diff
@@ -5,5 +5,8 @@
     "website": "https://lunapunks.io",
     "keybase": "https://keybase.io/hippietechie",
     "discord": "https://discord.gg/2DDXFCyAwe"
+  },
+  "notifications": { // notification settings
+      "email": "admin@lunapunks.io" // email to receive notifications
```

Comment:
JSON is a data-only format.  Please remove the comments and then we should be good to merge!